synapse/_scripts/synapse_port_db.py

```diff
@@ -167,22 +167,6 @@
     "ui_auth_sessions",
     "ui_auth_sessions_credentials",
     "ui_auth_sessions_ips",
-    # Groups/communities is no longer supported.
```

Comment:
I believe this is safe to remove since a user will have run the delta before running the port DB code?